MuonAnalysis/MomentumScaleCalibration/test/Macros/RooFit/MultiHistoOverlapAll_Z.C

```diff
@@ -331,7 +331,7 @@ void MultiHistoOverlapAll_Z(string files, string labels, string colors = "", str
       }
     }
     leg->Draw("same");
-    cmsPlotTitle->Draw("same");
+    //cmsPlotTitle->Draw("same");
```

Comment:
Hi,

 

These scripts are meant to be for cross checks only, so it could go either way. This criticism has come up several times.

 

I would suggest the person plotting these to give the decision. We only provide a template and a working example. IF we are to adjust these small things, we should also have better axis ranges etc.

 

Ulascan

 

 

 

From: Heshy Roskes [mailto:notifications@github.com] 
Sent: 27 June, 2016 13:13
To: cms-sw/cmssw <cmssw@noreply.github.com>
Cc: Ulascan Sarica <ulascan90@gmail.com>; Mention <mention@noreply.github.com>
Subject: Re: [cms-sw/cmssw] Fix DMR validation for phase 1 (#14997)

 

In MuonAnalysis/MomentumScaleCalibration/test/Macros/RooFit/MultiHistoOverlapAll_Z.C <https://github.com/cms-sw/cmssw/pull/14997#discussion_r68616646> :

> @@ -331,7 +331,7 @@ void MultiHistoOverlapAll_Z(string files, string labels, string colors = "", str
>        }
>      }
>      leg->Draw("same");
> -    cmsPlotTitle->Draw("same");
> +    //cmsPlotTitle->Draw("same");

@usarica <https://github.com/usarica>  
Just so you're aware, Victor mentioned at the alignment meeting a couple weeks ago (I think you weren't there) that he did not like having CMS preliminary on non-approved plots. Also there were some plots with other eta cuts so the |eta|<2.4 header did not make sense. I'm leaving the code here but commenting the Draw()

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/14997/files/dda5714ae7ced6d7df643bfaeb8e77f19ddae5ea#r68616646> , or mute the thread <https://github.com/notifications/unsubscribe/AFMvwMFNPeAy-LeLA2UleWnJr_VlNvf6ks5qQASYgaJpZM4I_TMd> .  <https://github.com/notifications/beacon/AFMvwDUTTuePo2Dw7N5gaM24x6j6aHybks5qQASYgaJpZM4I_TMd.gif>